website/docs/cloud-docs/run/remote-operations.mdx

```diff
@@ -22,10 +22,11 @@ Terraform runs managed by Terraform Cloud are called _remote operations._ Remote
 ### Disabling Remote Operations
 
 [execution_mode]: /cloud-docs/workspaces/settings#execution-mode
+[terraform_cloud_for_business]: https://cloud.hashicorp.com/docs/hcp/admin/billing/flex-billing#terraform-cloud-for-business
```

Comment:
PMM is suggesting to rename header to Terraform Cloud Business Tier

Reply:
https://github.com/hashicorp/cloud.hashicorp.com/pull/458